MAUI/Cartesian-Charts/Selection.md

```diff
@@ -84,44 +87,126 @@ Cartesian chart provides support to select a point programmatically on a chart u
+chart.Series.Add(series3);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Series Selection support in MAUI SfCartesianChart](Selection_images/maui_cartesianchart_series_selection.PNG)
+
+## Properties
+
+The following properties are common for both Selection Behaviors in [SfCartesianChart]() and [ChartSeries](), and the Selection feature can be configured using the following properties:
+
+* [Type]() - Gets or sets the ChartSelectionType Enum value for the Selection Behavior.     
+The following ChartSelectionType can be achieved during Selection:
+    * Single
+    * SingleDeselect
+    * Multiple
+    * None
+* [SelectionBrush]() - Gets or sets the SelectionBrush color value for the Selection Behavior.
+* [SelectedIndex]() - Gets or sets the index value of the segment or series that should be selected during the Selection.
+* [SelectedIndexes]() - Gets or sets the list of indexes of the segments or series that should be selected during the Selection.
+
+## Enable MultiSelection
+
+In [SfCartesianChart](), we can perform Multiple Selection for both DataPointSelection and SeriesSelection by selecting more than one segment in a series or more than one series in the chart by setting [SelectionBrush]() color and ChartSelectionType as Multiple, else by setting the [SelectedIndexes]() value from the ViewModel and set ChartSelectionType as Multiple to either [DataPointSelectionBehavior]() or [SeriesSelectionBehavior]() for load time selection changes.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+. . .
+    <chart:SfCartesianChart.BindingContext>
+        <model:ViewModel/>
+    </chart:SfCartesianChart.BindingContext>
+    <chart:SfCartesianChart.SelectionBehavior>
+        <chart:SeriesSelectionBehavior SelectionBrush="Green" Type = Multiple SelectedIndexes = "{Binding Indexes}" />
+    </chart:SfCartesianChart.SelectionBehavior>
+. . .
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+. . .
+ViewModel model = new ViewModel();
+chart.BindingContext = model;
+
+SeriesSelectionBehavior selection = new SeriesSelectionBehavior();
+selection.SelectionBrush = Brush.Green;
+selection.SelectedIndexes = model.Indexes;
+selection.Type = ChartSelectionType.Multiple;
+chart.SelectionBehavior = selection;
+. . .
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Multiple SeriesSelection support in MAUI SfCartesianChart](Selection_images/maui_multi_seriesselection.PNG)
+![Multiple DataPointSelection support in MAUI SfCartesianChart](Selection_images/maui_multi_dataPoint_selection.png)
+
+N> The Multiple Selection in [DataPointSelectionBehavior]() can also be performed by using the ChartSeries SelectionBehavior property.
+
+## ClearSelection Method
+
+Both the SelectionBehavior has a public method called [ClearSelection ()](), which resets all current Selection Behavior property values to their default values and resets the color of the selected element to default.
```

Comment:
Resets all the selection customization to default. The following code example shows how to use [ClearSelection]().

Reply:
Changed the content as mentioned.